Add video info (#145)
grafana/writers-toolkit: docs/sources/writing-guide/image-guidelines/index.md

   Store image files in the following website repo directory: `static/static/img/docs`. If a relevant sub-directory doesn't exist, you can create it.

1. After the Website team merges the image PR into the website repo, add the following figure shortcode to your docs:

   `{{</* figure src="[path to the image in the website repo]" */>}}`

   **Example:**
   - Path to image stored in the website repo: `static/static/img/docs/grafana-cloud/k8s-node-capacity.png`
   - Corresponding shortcode: `{{</* figure src="/static/img/docs/grafana-cloud/k8s-node-capacity.png" */>}}`.

1. Run `make docs` on your branch and verify that the image appears.
1. (Optional) Make adjustments to the image in the website repo and test again.

## Videos

> The Create Services team sometimes creates videos for blog posts and other collateral. Most of these videos are hosted on Vimeo.

Author: ```suggestion
The Creative Services team periodically creates videos for blog posts and other collateral. Most of these videos are hosted on Vimeo.
```

> You can embed a Vimeo-hosted video by using the `vimeo` short code and the video number:

Reviewer: Could you show the (unevaluated) shortcode for this example?  I guess you could just put
` {{< vimeo 773194063 >}}` 
on a line by itself like in the figure example in the previous section?

Author: Apparently, you can put the video link on it's own line or at the end of a sentence. It floats to where it fits.

> You can embed a Vimeo-hosted video by using the `vimeo` short code and the video number:

Author: ```suggestion
You can embed a Vimeo-hosted video by using the `vimeo` short code and the video number: `{{< vimeo 1111111>}}`.
```

> 
> ```
> {{< vimeo 773194063 >}}
> ```
> 
> In this example, the video is a Preview of Tempo 2.0 and TraceQL: `https://vimeo.com/773194063`
> 
> The video number at the end of the URL.

Author: ```suggestion

In this example, the video is a Preview of Tempo 2.0 and TraceQL: `https://vimeo.com/773194063`. The video number at the end of the URL.

```
{{< vimeo 773194063 >}}
```

```
